keyword/keyword.go

```diff
@@ -6,5 +6,6 @@ const (
 	Lang         = Keyword("LANG")
 	ImportRule   = Keyword("IMPORTRULE")
 	CannotImport = Keyword("CANNOTIMPORT")
+	CanImport    = Keyword("CANIMPORT")
```

Comment:
I would propose to rename `CANIMPORT` to `BUTALLOW`, `ALLOW` or `EXCEPT`

Reply:
I would prefer ALLOW. I will make the change.